types/react-native/index.d.ts

```diff
@@ -808,6 +809,14 @@ export interface TextPropsIOS {
      * default, a gray oval highlights the text on press down.
      */
     suppressHighlighting?: boolean;
+
+    /**
+     * Specifies largest possible scale a font can reach when allowFontScaling is enabled. Possible values:
+     * - null/undefined (default): inherit from the parent node or the global default (0)
+     * - 0: no max, ignore parent/global default
+     * - >= 1: sets the maxFontSizeMultiplier of this node to this value
+     */
+    maxFontSizeMultiplier?: number;
```

Comment:
This is missing the `null` typing

```suggestion
    maxFontSizeMultiplier?: number | null;
```